### PR TITLE
Build docs on RTD

### LINKED ISSUE #28: RTD does not build docs, defaults to Python 3.7 and Sphinx 1.8.6
RTD builds fail because it defaults to run Sphinx 1.8.6 on Python 3.7.

https://readthedocs.org/projects/plaster/builds/18689422/

### PR DESCRIPTION
**Pull request by stevepiercy**
Fixes https://github.com/Pylons/plaster/issues/28

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -66,7 +66,7 @@
 # the built documents.
 #
 # The short X.Y version.
-version = importlib.metadata.distribution("plaster").version
+version = pkg_resources.get_distribution("pyramid").version
```
Comment (stevepiercy) [inline docs/conf.py]:
```suggestion
version = pkg_resources.get_distribution("plaster").version
```

**Comment by stevepiercy** [comment]
All green now @mmerickel

**Comment by mmerickel** [comment]
The purpose of the change to use importlib.metadata was to stop using pkg_resources which requires setuptools. Please copy the pattern from https://github.com/Pylons/pastedeploy instead for this package.

We will need to add the importlib-metadata dependency to the docs extra on other repos as we move them away from setuptools. The alternative is to add setuptools to the docs extra so that you can import pkg_resources reliably.

**Comment by stevepiercy** [comment]
Changed, all green. Should I add a comment to remove the try/except when we drop Python 3.7?

**Comment by mmerickel** [comment]
No not necessary.

**Pull request merged**